openedx2zim/scraper.py

```diff
@@ -433,13 +620,20 @@ def run(self):
         )
         logger.debug("Checking for missing binaries")
         check_missing_binary(self.no_zim)
+        logger.info("Checking S3 cache credentials")
```

Comment:
This should be only printed if the arg was supplied

Reply:
Fixed